src/main/java/pl/javastart/task/model/Grade.java

```diff
@@ -0,0 +1,31 @@
+package pl.javastart.task.model;
+
+public class Grade {
+    private double grade;
+    private String groupCode;
+
```

Comment:
Ocena powinna łączyć informację o tym jaka jest ocena, z czego i kto ją dostał. Uprości to logikę w kilku miejscach Twojego kodu